src/grid/SaleGridView.php

```diff
@@ -161,11 +161,11 @@ public function columns()
                 'enableSorting' => false,
                 'format' => 'datetime',
             ],
-            'ticket' => [
-                'attribute' => 'ticket',
+            'reason' => [
+                'attribute' => 'reason',
                 'format' => 'raw',
-                'value' => fn(Sale $sale): string => $sale->ticket ? Html::a($sale->ticket,
-                    ['@ticket/view', 'id' => $sale->ticket],
+                'value' => fn(Sale $sale): string => $sale->reason ? Html::a($sale->reason,
+                    ['@ticket/view', 'id' => $sale->reason],
```

Comment:
Considering `reason` can be either internal integer ticket ID or a link to an external system, we should add conditional formatting.